src/index.css

```diff
@@ -1,3 +1,277 @@
+  max-width: 75rem;
+  margin-bottom: 1.813rem;
+}
+
+.hr hr {
+  border: 1px solid #969696;
+}
+
+.form {
+  padding-inline: 6.25rem;
+  max-width: 75rem;
+}
+
+.form h2 {
+  font-size: 1.5rem;
+  color: #c9c9c9;
+  font-weight: 900;
+  margin-bottom: 1.188rem;
+}
+
+.form input {
+  margin-right: 2.125rem;
+  background-color: white;
+  border: 1px solid #969696;
+  margin-bottom: 16.563rem;
+  padding-left: 1.063rem;
+  font-size: 1.188rem;
+  color: #969696;
+}
+
+input.title {
+  width: 35.688rem;
+  height: 2.813rem;
+}
+
+input.author {
+  width: 17.813rem;
+  height: 2.813rem;
+}
+
+.add-btn {
+  width: 11.5rem;
+  height: 2.813rem;
+  color: white;
+  background-color: #0290ff;
+  border: none;
+  font-size: 1.125rem;
+}
+
+.progress-btn {
+  height: 2.813rem;
+  color: white;
+  background-color: #0290ff;
+  border: none;
+  font-size: 1.125rem;
+  padding-left: 1.375rem;
+  padding-right: 1.188rem;
+}
+
+.add-btn:hover,
+.progress-btn:hover {
+  color: #0290ff;
+  background-color: white;
+  cursor: pointer;
+  border: 1px solid #0290ff;
+}
+
+.add-btn:active,
+.progress-btn:active {
+  color: white;
+  background-color: #0290ff;
+  cursor: pointer;
+}
+
+.book-list {
+  padding-inline: 6.25rem;
+  max-width: 75rem;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 3fr 2fr 2fr;
+  margin-bottom: 0.938rem;
+}
+
+.list {
+  height: 10.625rem;
+  margin-bottom: 0.938rem;
+  padding-left: 1.688rem;
+  padding-bottom: 1.75rem;
+  padding-top: 2rem;
+}
+
+div.list,
+div.btn-div,
+div.percentage,
+div.card {
+  background-color: white;
+}
+
+p.action {
+  font-size: 1.125rem;
+  background-color: white;
+  color: #c9c9c9;
+  margin-bottom: 0.25rem;
+}
+
+p.book-title {
+  background-color: white;
+  color: black;
+  font-weight: 900;
+  margin-bottom: 0.25rem;
+  font-size: 1.813rem;
+}
+
+p.book-author {
+  background-color: white;
+  color: #0290ff;
+  font-size: 1.188rem;
+  margin-bottom: 1.313rem;
+}
+
+.btn {
+  color: #0290ff;
+  background-color: white;
+  border: none;
+  font-size: 1.188rem;
+  padding-inline: 1.063rem;
+}
+
+.comment {
+  padding-left: 0;
+  border-right: 1px solid #969696;
+}
+
+.remove {
+  border-right: 1px solid #969696;
+}
+
+.percentage {
+  display: flex;
+}
+
+.percentage p,
+div.progress {
+  background-color: white;
+}
+
+p.number {
+  font-size: 2.438rem;
+  padding-top: 3.188rem;
+}
+
+p.completed {
+  font-size: 1.125rem;
+  color: #888;
+}
+
+div.middle {
+  margin-left: 0.688rem;
+  padding-bottom: 3.875rem;
+  background-color: white;
+}
+
+p.current {
+  font-size: 1.125rem;
+  color: #888;
+  background-color: white;
+  margin-top: 2.063rem;
+  margin-bottom: 0.438rem;
+}
+
+p.chapter {
+  background-color: white;
+  color: black;
+  font-size: 1.313rem;
+  margin-bottom: 1.438rem;
+}
+
+.remove:hover,
+.edit:hover,
+.comment:hover {
+  cursor: pointer;
+}
+
+.icon svg {
+  color: #0290ff;
+  font-size: 2.249rem;
+  background-color: white;
+}
+
+.circle svg {
+  color: #0290ff;
```

Comment:
- [ ] This should have color #4386bf. Please refer to the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information.